roles/etcd/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
 - name: Generate etcd certs
   include_tasks: "gen_certs_script.yml"
   when:
-    - cert_management | d('script') == "script"
+    - cert_management == "script"
```

Comment:
would it fails if cert_management is undefined ?

Reply:
Yeah, this would need a default value for cert_management in kubespray-defaults